### PR TITLE
Build code introspection service

### PR DESCRIPTION
**Pull request by thomtrp**
Starting to use ts-morph to retrieve function parameters

### CODE REVIEW

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.module.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
```
Comment (martmull) [inline packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.module.ts:1]: think this should go to `modules` in a dedicated `modules/code-introspection` waiting for the serverless function migration to be done into `modules` (scared of conflicts). I will move that then

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.service.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+@Injectable()
+export class CodeIntrospectionService {
+  private project: Project;
+
+  constructor() {
+    this.project = new Project();
+  }
+
+  public analyze(
```
Comment (martmull) [inline packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.service.ts:28]: would be great to have some testing, to see quickly how this analyse function works

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.module.ts (hunk 1 above)
Comment (thomtrp) [reply packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.module.ts]: done

File: packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.service.ts (hunk 2 above)
Comment (thomtrp) [reply packages/twenty-server/src/engine/metadata-modules/serverless-function/code-introspection/code-introspection.service.ts]: done

**Review by martmull — APPROVED** [review]
LGTM

**Pull request merged**